source/chapters/library.rst

```diff
@@ -475,6 +474,28 @@ Mixxx supports the following filters:
      You can put a space after the colon but currently there must be no space
      between the operator and the number.
 
+In addition to combining filters directly (effectively an "AND"), you can also combine queries using the "OR" operator, spelled ``|``.
+
+Examples
+  ::
+  
+    genre:house | genre:techno
+
+  The "OR" operator has a lower precedence than regular filters, so you can create larger queries too. The following query, for example, would match all tracks that are 5-star Hip-Hop or Pop:
```

Comment:
I'm not sure if that means
[5-star Hip-hop] | Pop
or
5-star [Hip-hop | Pop]
Can we rephrase it
```suggestion
  The "OR" operator has a lower precedence than regular filters, so you can create larger queries too. The following query, for example, would match all 5-star tracks of either Hip-Hop or Pop genre:
```

Reply:
It it supposed to mean `[5-star Hip-hop] | Pop` (since the OR binds more loosely than AND) and I agree, it does feel a bit ambiguous. Not sure how to phrase it better though, I feel like the suggestion leans itself more towards the second interpretation.

Note that the second one cannot actually be expressed with the current machinery since we don't have any parentheses yet (if I haven't missed something).

Reply:
Perhaps simply flipping the operands would make it more clear:

```suggestion
  The "OR" operator has a lower precedence than regular filters, so you can create larger queries too. The following query, for example, would match all tracks that are Pop or 5-star Hip-Hop:
```